common/changes/sparo/feat-sparo-branch_2024-05-21-23-18.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "sparo",
+      "comment": "Add a renamed command \"sparo git-branch\"",
```

```suggestion
      "comment": "Add a mirrored command \"sparo git-branch\"",
```

The docs call them [mirrored commands](https://tiktok.github.io/sparo/pages/commands/overview/)